cli/src/command.rs

```diff
@@ -91,7 +91,15 @@ pub enum InfoCommand {
     /// get logs from a build
     Logs {
         /// id of the build, latest if empty
-        id: Option<String>
+        id: Option<String>,
+
+        /// subscribe to live logs
+        #[clap(short, long)]
+        subscribe: Option<bool>,
+
+        /// stay attached to live logs
+        #[clap(short, long, requires("subscribe"))]
+        attach: Option<bool>
```

Comment:
I'd rename this option to something different, as attach is to me synonymous to subscribe. I'm thinking of something like `--keep`, `--stay` or `--linger`, just something that indicates that we stay subscribed in one word. We could even rename the subscribe flag to attach then.

Reply:
And we should also make these options a bool, not an option, so we d'ont have to specify `--subscribe true` but just `--subscribe`.